BREEZE.rst

```diff
@@ -806,32 +718,6 @@ through ``breeze testing docker-compose-tests`` command.
 
 The docker-compose tests are in ``docker-tests/`` folder in the main repo.
 
-Running Kubernetes tests
-........................
-
-Breeze helps with running Kubernetes tests in the same environment/way as CI tests are run.
-Breeze helps to setup KinD cluster for testing, setting up virtualenv and downloads the right tools
-automatically to run the tests.
-
-You can:
-
-* Manage KinD Kubernetes cluster and deploy Airflow to KinD cluster ``./breeze-legacy kind-cluster`` commands
-* Run Kubernetes tests  specified with ``./breeze-legacy kind-cluster tests`` command
-* Enter the interactive kubernetes test environment with ``./breeze-legacy kind-cluster shell`` command
-
-This is described in detail in `Testing Kubernetes <TESTING.rst#running-tests-with-kubernetes>`_.
```

Comment:
Did you mean to remove this section?

Reply:
Yep. This is very temporary and it is going to be replaced by #25678